packages/workflow-compile/index.js

```diff
@@ -82,6 +84,17 @@ const WorkflowCompile = {
         compilers
       });
     }
+
+    if (
+      config.db &&
+      config.db.enabled === true &&
+      contracts.length > 0 &&
+      loadDb === true
+    ) {
+      const db = new TruffleDB(config);
+      await db.loadCompilations({ contracts, compilations });
+    }
```

Comment:
This logic should get moved to `save`, since that's distinct from compilation.

Reply:
moved